garrysmod/lua/includes/modules/player_manager.lua

```diff
@@ -24,12 +24,28 @@ function AddValidModel( name, model )
 
 end
 
+
+
+--[[---------------------------------------------------------
+	Helper function as GetBodyGroups is a table
+-----------------------------------------------------------]]
+function GetBodyGroupsAsStr( ply )

```

Comment:
I think this should be moved to `lua/includes/extensions/entity.lua` as an entity extension, to complement https://wiki.facepunch.com/gmod/Entity:SetBodyGroups

The code should also conform to https://github.com/Facepunch/garrysmod/blob/master/CONTRIBUTING.md aka be consistent with the rest of the code base.

Also, should be named `Entity:GetBodyGroupsAsString`, `Entity:GetBodyGroupString` or `Entity:GetActiveBodyGroups`